### ISSUE TITLE
measure complexity

### ISSUE THREAD

**Issue by lacava** [EpistasisLab/regression-benchmark#3]
we should measure complexity of the final models produced by the methods. Since they all vary quite a bit, this isn't trivial. for the SR methods, we should be able to count the number of nodes (operators and literals) in the solutions.

**Comment by lacava** [comment]
@jmmcd for FFXRegressor, is the # of nodes in the GP-equivalent tree given by `self._models[-1].complexity()` ? If not can you define it for us?

**Comment by jmmcd** [comment]
Yes, that's right, see https://github.com/natekupp/ffx/blob/77cc110960828dcf66e0462170319bc69a8ce34c/ffx/core.py#L285

**Comment by foolnotion** [comment]
In our trees, variable nodes have an associated weight, so that the output is always weight x variable value (from the dataset). Currently, these are not counted towards the complexity (e.g., a variable node has complexity 1). 

Should we change this to take the multiplication into account? Such that `complexity(variable node) == complexity (Multiply(variable node, constant node)))` = 3 ?

**Comment by lacava** [comment]
@foolnotion yes, the multiplication should count. I did something similar for [FEAT](https://github.com/EpistasisLab/srbench/blob/89d43454940eda8fc59489b50b5bdf28a397d228/experiment/methods/FEATRegressor.py#L30)
